README.md

```diff
@@ -16,7 +16,7 @@ own network).
 At this time, WebREPL client cannot be accessed over HTTPS connection.
 This is due to not widely published policy that HTTPS pages may
 access only WSS (WebSocket Secure) protocol. This is somewhat
-similar to warnings issues when e.g. an HTTPS paeg loads an image
+similar to warnings issues when e.g. a HTTPS page loads an image
```

Comment:
IMO it should be "an HTTPS page" because you pronounce it as "aich tee tee pee ess", see eg http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html